types/react-mathquill/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for react-mathquill 0.1
+// Project: https://github.com/viktorstrate/react-mathquill#readme
+// Definitions by: Marco Gonzalez <https://github.com/magonzalez9>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { ComponentProps, ReactNode, Component } from 'react';
+import { MQ, Config } from 'edtr-io__mathquill';
+
+export interface Props extends Omit<ComponentProps<'span'>, 'onChange'> {
+    latex: string;
```

Comment:
This field is [optional].

```suggestion
    latex?: string;
```

[optional]: https://github.com/viktorstrate/react-mathquill/blob/4da4a8a95f4b092e79ec9a522957a98bc3f3b137/src/Component.js#L46